examples/oven-app/silabs/src/OvenManager.cpp

```diff
@@ -45,27 +45,17 @@ void OvenManager::Init()
 {
     DeviceLayer::PlatformMgr().LockChipStack();
 
-    // Initialize states
-    mCookTopState      = kCookTopState_Off;
-    mCookSurfaceState1 = kCookSurfaceState_Off;
-    mCookSurfaceState2 = kCookSurfaceState_Off;
-
     // Endpoint initializations
-    VerifyOrReturn(mTemperatureControlledCabinetEndpoint.Init() == CHIP_NO_ERROR,
-                   ChipLogError(AppServer, "TemperatureControlledCabinetEndpoint Init failed"));
+    VerifyOrDie(mTemperatureControlledCabinetEndpoint.Init() == CHIP_NO_ERROR);
 
     // Set initial state for TemperatureControlledCabinetEndpoint
-    VerifyOrReturn(SetTemperatureControlledCabinetInitialState(kTemperatureControlledCabinetEndpoint) == CHIP_NO_ERROR,
-                   ChipLogError(AppServer, "SetTemperatureControlledCabinetInitialState failed"));
-
+    VerifyOrDie(SetTemperatureControlledCabinetInitialState(kTemperatureControlledCabinetEndpoint) == CHIP_NO_ERROR);
     // Register the shared TemperatureLevelsDelegate for all the cooksurface endpoints
     TemperatureControl::SetInstance(&mTemperatureControlDelegate);
 
     // Set initial state for CookSurface endpoints
-    VerifyOrReturn(SetCookSurfaceInitialState(mCookSurfaceEndpoint1.GetEndpointId()) == CHIP_NO_ERROR,
-                   ChipLogError(AppServer, "SetCookSurfaceInitialState failed for CookSurfaceEndpoint1"));
-    VerifyOrReturn(SetCookSurfaceInitialState(mCookSurfaceEndpoint2.GetEndpointId()) == CHIP_NO_ERROR,
-                   ChipLogError(AppServer, "SetCookSurfaceInitialState failed for CookSurfaceEndpoint2"));
+    VerifyOrDie(SetCookSurfaceInitialState(mCookSurfaceEndpoint1.GetEndpointId()) == CHIP_NO_ERROR);
+    VerifyOrDie(SetCookSurfaceInitialState(mCookSurfaceEndpoint2.GetEndpointId()) == CHIP_NO_ERROR);
```

Comment:
That's a lot of `VerifyOrDie` We are sure that we can't recover if it fails and that the only solution is to stop everything ?